ground/gcs/src/plugins/rawhid/rawhidplugin.h

```diff
@@ -95,7 +92,7 @@ class RAWHID_EXPORT RawHIDPlugin
     virtual bool initialize(const QStringList &arguments, QString *error_message);
     virtual void extensionsInitialized();
 private:
-    RawHIDConnection *hidConnection;
+    RawHIDConnection* hidConnection;
```

Comment:
Ah.  I think someone @ Tau asked me to do it the other way.  That's a relief.

I will make this PR mostly conform to the right way and follow the right way goin' forward.